extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -54,11 +54,22 @@ The `amp-subscriptions` extensions implements the subscription-style access/payw
 
 ## Relationship to `amp-access`
 
-The `amp-subscriptions` is similar to [`amp-access`](../amp-access/amp-access.md) and in many features builds on top of `amp-access`. However, it's a much more specialized version of access/paywall protocol. Some of the key differences are:
+The `amp-subscriptions` is similar to [`amp-access`](../amp-access/amp-access.md)
+and in many features builds on top of `amp-access`. However, it's a much more
+specialized version of access/paywall protocol. Some of the key differences are:
 
-1. Entitlements response is similar to the amp-access authorization, but it's striclty defined and standardized.
-2. The `amp-subscriptions` allows multiple services to be configured for the page to participate in access/paywall decisions. They are executed in parallel and paralized based on which service returns the positive response.
-3. The viewers are allowed to provide a signed authorization response based on an independent agreement with publishers as a proof of access.
+1. The `amp-subscriptions` entitlements response is similar to the amp-access
+authorization, but it's striclty defined and standardized.
+2. The `amp-subscriptions` extension allows multiple services to be configured
+for the page to participate in access/paywall decisions. They are executed
+concurrently and prioritized based on which service returns the positive response.
+3. AMP viewers are allowed to provide a signed authorization response based
```

Comment:
Can this be clarified as true for `amp-subscriptions` or `amp-access`? 

Same for number 4.